HunterPie.UI/Overlay/Widgets/Activities/Rise/MeowcenariesContextHandler.cs

```diff
@@ -20,6 +21,9 @@ public MeowcenariesContextHandler(MHRContext context)
         {
             _context = context;
             _player = context.Game.Player as MHRPlayer;
+            ViewModel.StepString = Localization.QueryString("//Strings/Client/Overlay/String[@Id='MEOWCENARIES_STEP_STRING']");
+            ViewModel.OutcomeString = Localization.QueryString("//Strings/Client/Overlay/String[@Id='MEOWCENARIES_OUTCOME_STRING']");
+            ViewModel.BuddyString = Localization.QueryString("//Strings/Client/Overlay/String[@Id='MEOWCENARIES_BUDDY_STRING']");
```

Comment:
Instead of using the `ViewModel` to control the localization, you can use the [LocalizationString](https://github.com/Haato3o/HunterPie-v2/blob/42c40ffdf31032b82372601206d47efa0deb9478/HunterPie.UI/Architecture/Assets/GetAsset.cs#L39) markup extension to query the string directly from the `View` itself.

Here's an example on how to use it:
https://github.com/Haato3o/HunterPie-v2/blob/42c40ffdf31032b82372601206d47efa0deb9478/HunterPie/MainWindow.xaml#L87-L90

Don't forget to import that namespace by putting `xmlns:loc="clr-namespace:HunterPie.UI.Architecture.Assets"` in the header of the file.